server/routes_delete_test.go

```diff
@@ -33,7 +33,8 @@ func TestDelete(t *testing.T) {
 
 	w = createRequest(t, s.CreateHandler, api.CreateRequest{
 		Name:      "test2",
-		Modelfile: fmt.Sprintf("FROM %s\nTEMPLATE {{ .System }} {{ .Prompt }}", createBinFile(t, nil, nil)),
+		FromModel: &api.CreateFromModel{Type: "gguf", Files: map[string]string{"test.gguf": digest}},
```

Comment:
Similar comment above, but `FromModel` is a bit confusing of name. Is there a way this could just become:

```
FromFiles: map[string]string{"test.gguf": digest
```

or even

```
From: map[string]string{"test.gguf": digest}
```

The type overloading is a bit painful to work with so I'm ok with us keeping distinctive fields instead